jsonschema-maven-plugin/src/test/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojoTest.java

```diff
@@ -113,6 +113,97 @@ public void testPomConfigurationErrors(String testCaseName) throws Exception {
         executePom(new File(testCaseLocation, testCaseName + "-pom.xml"));
     }
 
+    /**
+     * Unit test to test the generation of schemas for multiple classes
+     */
+    @Test
+    public void testTwoClasses() throws Exception {
+        File testCaseLocation = new File("src/test/resources/reference-test-cases");
+        File generationLocation = new File("target/generated-test-sources/TwoClasses");
+
+        // Execute the pom
+        executePom(new File("src/test/resources/reference-test-cases/TwoClasses-pom.xml"));
+
+        // Validate that the schema files are created.
+        File resultFileA = new File(generationLocation,"TestClassA.schema");
+        Assert.assertTrue(resultFileA.exists());
+
+        File resultFileB = new File(generationLocation,"TestClassB.schema");
+        Assert.assertTrue(resultFileB.exists());
+
+        // Validate that they are the same as the reference
+        File referenceFileA = new File(testCaseLocation + "/" + "TestClassA-reference.json");
+        Assert.assertTrue(referenceFileA.exists());
+        Assert.assertTrue("Generated schema for TestClassA is not equal to the expected reference.",
+                FileUtils.contentEquals(resultFileA, referenceFileA));
```

Comment:
In the other parts of this project, I've used `JSONAssert` to compare expected and generated schemas.

But I can align that myself before the final merge.

Reply:
Can you make this change? Thank you.